### ISSUE TITLE
[Transfered issue]

### ISSUE THREAD

**Issue by mentosti** [NECROBOTIO/NecroBot#4325]
Hi,
I don't know whether or not this is the problem.
But when i saw console, i noticed that the program transfered some pokemons with higher CP than the lower one.
And this is the snapshot i captured.
Please take a look.
Thank you.
![untitled](https://cloud.githubusercontent.com/assets/7583753/17544155/ce27cba2-5eff-11e6-8487-9e99d54ff476.png)

**Comment by theloser** [comment]
You do have PrioritizeIvOverCp set to true, right?

**Comment by Yummster** [comment]
it transferred a Draitini with 59.81% IV and informed you your best one has 98.24%

**Comment by mentosti** [comment]
i keep the default settings, and it's default value is false.

**Comment by theloser** [comment]
You sure? Because it looks like the boot kept the pokemon with the higher IV.

**Comment by mentosti** [comment]
Here is my config file.
I didn't change any value, except lat, long
![image](https://cloud.githubusercontent.com/assets/7583753/17544348/cf4f1886-5f00-11e6-9942-9acd2f131845.png)

**Comment by AlucardDemon** [comment]
I just had the same thing happen too. Mine is also set to false and it got rid of a pinsir with 850 CP and the lowest I have is 375

**Comment by Schweng** [comment]
Yep, same here. Transferred a 1093 Omot with 51%, because i have a 140 Omot with 92%...

PrioritizeIvOverCp is false.

**Comment by engharat** [comment]
It just happened to me too. "KeepMinDuplicatePokemon" : 3, but it started stransferring ALL my pokemons. I just closed it  before losing the most valuable ones. I then checked the logs and it transferred a bunch of pokemons that were in 1 copy.

**Comment by mentosti** [comment]
i met the problem sometimes, here is another one.
As you can see.
the current highest CP of eevee is 639, but i saw in logs. The program captured another eevee with higher CP, but it was transfered
![untitled](https://cloud.githubusercontent.com/assets/7583753/17544586/74e15a60-5f02-11e6-9ee3-860f21833fbe.png)
![image](https://cloud.githubusercontent.com/assets/7583753/17544669/d4d633dc-5f02-11e6-9798-51a8adc35c93.png)

**Comment by engharat** [comment]
logs:
`[08:43:38] Charmeleon	- CP: 878  IV: 22.26%   [Best CP: 878  IV: 22.26%] (Candies: 11)
[08:43:39] Graveler	- CP: 909  IV: 82.02%   [Best CP: 909  IV: 82.02%] (Candies: 19)
[08:43:40] Machoke	- CP: 970  IV: 79.49%   [Best CP: 970  IV: 79.49%] (Candies: 32)
[08:43:42] Ponyta	- CP: 844  IV: 86.53%   [Best CP: 844  IV: 86.53%] (Candies: 33)
[08:43:44] Poliwrath	- CP: 987  IV: 44.33%   [Best CP: 987  IV: 44.33%] (Candies: 11)
[08:43:45] Marowak	- CP: 773  IV: 77.99%   [Best CP: 773  IV: 77.99%] (Candies: 29)
[08:43:46] Pidgeotto	- CP: 982  IV: 68.63%   [Best CP: 982  IV: 68.63%] (Candies: 6)
[08:43:48] Weepinbell	- CP: 910  IV: 57.42%   [Best CP: 910  IV: 57.42%] (Candies: 23)
[08:43:49] Magmar	- CP: 223  IV: 62.16%   [Best CP: 223  IV: 62.16%] (Candies: 4)
[08:43:51] Hitmonlee	- CP: 773  IV: 75.89%   [Best CP: 773  IV: 75.89%] (Candies: 8)
[08:43:52] Slowpoke	- CP: 278  IV: 48.29%   [Best CP: 278  IV: 48.29%] (Candies: 32)
[08:43:54] Seel	- CP: 598  IV: 74.49%   [Best CP: 598  IV: 74.49%] (Candies: 22)
[08:43:55] Growlithe	- CP: 17  IV: 69.40%   [Best CP: 17  IV: 69.40%] (Candies: 2)
[08:43:57] Cubone	- CP: 43  IV: 93.52%   [Best CP: 43  IV: 93.52%] (Candies: 29)
[08:43:58] Oddish	- CP: 14  IV: 41.65%   [Best CP: 651  IV: 96.73%] (Candies: 95)
[08:44:00] Paras	- CP: 328  IV: 85.16%   [Best CP: 328  IV: 85.16%] (Candies: 17)
[08:44:01] Mankey	- CP: 137  IV: 77.38%   [Best CP: 137  IV: 77.38%] (Candies: 36)`

Config:
`"KeepPokemonsThatCanEvolve": true,
  "KeepMinCp": 800,
  "KeepMinIvPercentage": 95.0,
  "KeepMinLvl": 6,
  "KeepMinOperator": "or",
  "UseKeepMinLvl": false,
  "PrioritizeIvOverCp": false,
  "KeepMinDuplicatePokemon": 3,`

**Comment by engharat** [comment]
Moreover,with pogofeeder I just got my first snorlax (no duplicate), and snorlax is in the list of PokemonsNotToTransfer, so it has 2 goods reason to not transfer it , and still it transferred it!
I think it could a bug of the config.json itself, something about not prioritizing the keeping of duplicates and the PokemonsNotToTransfer list?
Because I've kept another config.json for backup, the one that I use to farm exp only, I changed the few lines needed to work with pogolocationfeeder and now is working properly,without trashing my single copy pokemon - and I'm still using the same necrobot, 0.7.6 version.
